html/elements/script.json

```diff
@@ -602,10 +605,12 @@
                   "version_added": false
                 },
                 "opera": {
```

Comment:
(I'm assuming that Opera, Samsung, and WebView are the same as Chrome.)

Reply:
I would *like* to just presume the same applies to Opera, Samsung, and WebView, but, it seems that caniuse.com and MDN have a discrepancy regarding Opera's module support, and I remember quite literally, telling someone that WebView didn't support modules only a few months ago, maybe I was misinformed?

Regardless, it's probably safe to make the assumption.